packages/components/src/logos/jetpack-logo/index.tsx

```diff
@@ -81,7 +88,6 @@ const JetpackLogo: React.FunctionComponent< JetpackLogoProps > = ( {
 
 	if ( 24 === size ) {
```

Comment:
There is a bug where `<JetpackLogo size={ 24 } />` renders in monochrome. There is also some apparent redundancy in how monochrome is handled within the component structure. Will fix separately.

Reply:
This might have been intentional back in the day, but I agree it's odd and should be fixed so it's deliberate in the consumer land, and not here.